users/models.py

```diff
@@ -1,95 +1,99 @@
+        MOM = 0  # 엄마
+        DAD = 1  # 아빠
+        BROTHER1 = 2  # 형제1
+        BROTHER2 = 3  # 형제2
+        BROTHER3 = 4  # 형제3
 
-    id = models.PositiveIntegerField(primary_key=True)          #id
-    type = models.IntegerField(choices = CharacterType.choices) #캐릭터 이미지
+    id = models.PositiveIntegerField(primary_key=True)  # id
+    type = models.IntegerField(choices=CharacterType.choices)  # 캐릭터 이미지
 
 
 # 유저 정보 테이블
 class Info(accounts.models.BaseModel):
-    class ActivityType(models.IntegerChoices):      #활동수준
-        NOCHOICES = -1  #선택을 안 했을 때(비 당뇨인의 경우)
-        DEEPLOW = 0     #활동이 적거나 운동을 안하는 경우
-        LOW = 1         #가벼운 활동 및 운동
-        MIDDLE = 2      #보통의 활동 및 운동
-        HIGH = 3        #적극적인 활동 및 운동
-        DEEPHIGH = 4    #매우 적극적인 활동 및 운동
-
-    class GenderType(models.TextChoices):           #성별
-            FEMALE = 'f'
-            MALE = 'm'
-
-    # id                                                                                            #로그인테이블 ID
-    name = models.CharField(max_length=20)                                                          #이름
-    character_id = models.ForeignKey(Character, on_delete=models.SET_NULL, null=True)               #캐릭터 ID
-    height = models.FloatField(default=None, null=True)                                             #키
-    weight = models.FloatField(default=None, null=True)                                             #몸무게
-    gender = models.CharField(max_length=1, choices = GenderType.choices)                           #성별
-    is_diabetes = models.BooleanField(default=False)                                                #당뇨환자여부
-    activity = models.IntegerField(default=ActivityType.NOCHOICES, choices = ActivityType.choices)  #활동수준
-    # account_user_group_id                                                                         #그룹코드 ID
+    class ActivityType(models.IntegerChoices):  # 활동수준
+        NOCHOICES = -1  # 선택을 안 했을 때(비 당뇨인의 경우)
+        DEEPLOW = 0  # 활동이 적거나 운동을 안하는 경우
+        LOW = 1  # 가벼운 활동 및 운동
+        MIDDLE = 2  # 보통의 활동 및 운동
+        HIGH = 3  # 적극적인 활동 및 운동
+        DEEPHIGH = 4  # 매우 적극적인 활동 및 운동
+
+    class GenderType(models.TextChoices):  # 성별
+        FEMALE = 'f'
+        MALE = 'm'
+
+    id = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)  # 로그인테이블 ID
+    name = models.CharField(max_length=20)  # 이름
+    character_id = models.ForeignKey(Character, on_delete=models.SET_NULL, null=True)  # 캐릭터 ID
+    height = models.FloatField(default=None, null=True)  # 키
+    weight = models.FloatField(default=None, null=True)  # 몸무게
+    gender = models.CharField(max_length=1, choices=GenderType.choices)  # 성별
+    is_diabetes = models.BooleanField(default=False)  # 당뇨환자여부
+    activity = models.IntegerField(default=ActivityType.NOCHOICES, choices=ActivityType.choices)  # 활동수준
+    account_user_group_id = models.ForeignKey(Group, on_delete=models.SET_NULL, null=True)  # 그룹코드 ID
```

Comment:
```suggestion
    group = models.ForeignKey(Group, on_delete=models.SET_NULL, null=True)  # 그룹코드 ID
```
이 부분도 위와 마찬가지의 이유로 이름에 id없어도 될 것 같아요! 더불어 저희가 원래는 account앱에 그룹테이블을 생성하려 했는데 현재는 user 앱에 생성했으니, account_user도 제외해도 될 것 같습니다!